code/frameworks/nextjs/template/next-env.d.ts

```diff
@@ -0,0 +1,7 @@
+// Reference necessary since Next.js 13.2.0, because types in `next/navigation` are not exported per default, but
```

Comment:
Will this file only be included in Next 13 projects?

Reply:
It will not be included at all. It is to satisfy our internal type checks.

Reply:
`next-env.d.ts` is generated by Next.js during startup. We don't have to include this for the sandboxes, because the content varies upon how you configure Next.js (with or without app directory)

Reply:
Wouldn't [this](https://github.com/storybookjs/storybook/blob/next/code/lib/cli/src/helpers.ts#L186) make it so that this file will be added to users' projects?

Reply:
No. The `templateDir` is not `template`, but a subfolder of it

Reply:
Alright then! Thanks for explaining